tests/mongodb_import_connect.robot

```diff
@@ -8,27 +8,27 @@ Suite Setup         Set Library Search Order    SeleniumLibrary
 Suite Teardown      Tear Down The Test Suite
 Test Setup          Given The Browser Is On Openshift Home Screen
 Test Teardown       Close Browser
-
+Force Tags          Mongo
```

Comment:
This Force tag used for test execution exclusion on local, It would be removed/enhanced on upcoming PR